### ISSUE TITLE
List completely empties itself when offline

### ISSUE THREAD

**Issue by Midtan** [erengy/taiga#739]
So I decided to try out Kitsu since I'm fed up with waiting for MAL to to anything and set up Taiga to sync with Kitsu.
Since I did that, whenever I start Taiga in offline mode, my list is completely empty until I go online and sync it with Kitsu again. I turned off the option to sync at startup but that didn't help and as far as I remember was ticked when using Taiga with MAL as well. I never had said problem when it was still set to sync with MAL and I certainly would like to be able to use Taiga like I did before.

I am using Taiga 1.3.1

**Comment by Midtan** [comment]
Update:

I checked Taigas data directory and there seem to be no differences between the db and list files before closing, after closing and before opening when offline. What I did notice was that there still is the old folder for my MAL list which I imagine should not really cause a problem. Furthermore the directory in which my Kitsu list is stored is missing a user name, but it does that while online as well.
